authorization/src/main/kotlin/com/ritense/authorization/ValtimoAuthorizationService.kt

```diff
@@ -116,6 +124,21 @@ class ValtimoAuthorizationService(
             }
     }
 
+    private fun logPermissions(request: AuthorizationRequest<*>, permissions: List<Permission>? = null) {
+        if (request.action.key == Action.DENY) {
+            logger.error {
+                "Access denied on '${request.resourceType}'. This generally indicates attempting to " +
+                    "access a resource without considering authorization. Please refer to the Valtimo documentation."
+            }
+        } else {
+            val userPermissions = permissions ?: getPermissions(request)
```

Comment:
@theo-ritense Its only logging the applied permissions. But it does log a lot. For example when assigning a 'developer' to a task, it logs:
```
User 'admin@example.com' is requesting permission to 'view:CamundaTask' and has the following matching permissions: [b1108a2d-66b0-4abf-bcd3-76760c6dafbd:ROLE_USER]
User 'admin@example.com' is requesting permission to 'view:CamundaTask' and has the following matching permissions: [b1108a2d-66b0-4abf-bcd3-76760c6dafbd:ROLE_USER]
User 'admin@example.com' is requesting permission to 'assign:CamundaTask' and has the following matching permissions: [16713494-e9f4-4c25-9bdf-f4c93030ef9d:ROLE_ADMIN]
User 'developer@example.com' is requesting permission to 'assignable:CamundaTask' and has the following matching permissions: [02fb3459-6248-4b5b-a72b-57c1cb7a2594:ROLE_USER]
```